Add "Notifications" section and add the news (#266)
sider/sider-docs: docs/news/2019.md

This document describes notable changes on Sider in the year 2019.

## Configure Notifications for Repositories

<time class="news-date" datetime="2019-11-20">November 20, 2019</time>

> We have released a feature in which you can configure notifications on Sider analysis.

Reviewer: I don't think "we can configure notifications 'in a feature.'"
How about this one?

```suggestion
We have released a feature that will let you configure notifications on Sider analysis.
```

Author: Thanks! I've applied 🆗
